.eslintrc.js

```diff
@@ -59,49 +65,54 @@ module.exports = {
         'prefer-const': ['error', {}],
         'react/jsx-closing-bracket-location': ['error', {
             'nonEmpty': 'after-props',
-            'selfClosing': 'line-aligned',
+            'selfClosing': 'line-aligned'
```

Comment:
I'm not sure where the policy on dangling commas changed. Was this deliberate? I much prefer dangling commas, and the fact that it could change here tells me that we should introduce a linter rule for dangling commas.

Can you add that to this PR as well?
https://eslint.org/docs/rules/comma-dangle

Reply:
Okay, will do!